daisy/rocky/8/rocky_linux_8_arm64.wf.json

```diff
@@ -0,0 +1,56 @@
+{
+  "Name": "build-rocky-linux-8-arm64",
+  "Vars": {
+    "google_cloud_repo": {
+      "Value": "stable",
+      "Description": "The Google Cloud Repo branch to use."
+    },
+    "installer_iso": {
+      "Required": true,
+      "Description": "The Rocky Linux 8 installer ISO to build from."
+    },
+    "build_date": {
+      "Value": "${TIMESTAMP}",
+      "Description": "Build datestamp used to version the image."
+    },
+    "publish_project": {
+      "Value": "${PROJECT}",
+      "Description": "A project to publish the resulting image to."
+    }
+  },
+  "Steps": {
+    "build-rocky": {
+      "Timeout": "60m",
+      "IncludeWorkflow": {
+        "Path": "./enterprise_linux.wf.json",
+        "Vars": {
+          "el_release": "rocky-linux-8-arm64",
+          "kickstart_config": "./kickstart/rocky_linux_8_arm64.cfg",
+          "installer_iso": "${installer_iso}",
+          "machine_type": "t2a-standard-4",
```

Comment:
This will work, but using c4a can be beneficial due to better hardware availablity.

Reply:
Done!